pennylane/qaoa/mixers.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+This file contains built-in functions for defining QAOA workflows
+"""
+import networkx
+import pennylane as qml
+from pennylane.wires import Wires
+
+
+def x_mixer(n):
+    r"""Creates the basic Pauli-X mixer Hamiltonian used in the original QAOA paper.
+
+    This Hamiltonian is defined as:
+
+    .. math:: H_M \ = \ \displaystyle\sum_{i} X_{i},
+
+    where :math:`i` ranges over all wires, and :math:`X_i` denotes the Pauli-X operator on the :math:`i`-th wire.
+
+    Args:
+        n (int): The number of wires on which the Hamiltonian is applied
+
+    Returns:
+        type: description
+        ~.Hamiltonian:
```

Comment:
```suggestion
        Hamiltonian: the X-mixer Hamiltonian
```
Actually removing the tilde and dot made it appear as a nice link:
![image](https://user-images.githubusercontent.com/49409390/88305382-6b646d00-ccd7-11ea-81cb-19d203893b7c.png)

Reply:
Also good to add some text after the colon

Reply:
@trbromley I suggested removing the text after the colon :) The reason being is for small functions like this, the function summary essentially just duplicates the return details.

I've seen this in other libraries as well. My rule of thumb tends to be: If the docstring summary describes the return (e.g., it is of the form 'Returns ...'), then no need to add a redundant return description.